### ISSUE TITLE
As a developer, I would like to spend less time smoketesting cellxgene deployments and have confidence that the smoketests will catch user experience regressions.

### ISSUE THREAD

**Issue by maniarathi** [chanzuckerberg/single-cell#50]
**Story**: As a developer, I would like to avoid spending 30 minutes to 1 hour every week manually testing the cellxgene staging environment prior to release. Manual testing is error prone (this bug wasn't caught for nearly 1 month) and time consuming. As a developer, I'd like to have confidence in my smoke tests to capture user interactions more thoroughly and consistently so that we have fewer regressions and few aborted deployments.
**Need**: ~8 engineering hours per week are spent towards smoke-testing cellxgene deployments. This time could be better spent elsewhere while an automated smoketest runs to provide more consistent results on whether the latest cellxgene deployment has any user experience regressions.

**TL;DR**: Add the red colored use cases in the QA script to the cellxgene smoketest suite.

**Comment by maniarathi** [comment]
This epic will be supplanted by introducing image diff-ing through Percy, so closing.
